docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+## Creating the ASP.NET Core 2.1 Project using Visual Studio 2017
+
+ASP.NET Core is a general-purpose development platform maintained by Microsoft and the .NET community on GitHub. It is cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and embedded/IoT scenarios.
+
+For this example, we use a simple project based on a Visual Studio Web API template, so you don't need any additional knowledge to create the sample, you only have to create the project using a standard template that includes all the elements to run a small project with a REST API, using ASP.NET Core 2.1 technology.
+
+![Add new project window in Visual Studio, selecting ASP.NET Core Web Application.](media/create-aspnet-core-application.png)
+
+**Figure 4-36**. Creating ASP.NET Core Application
+
+To create the sample project, you have to select **New > Project** on Visual Studio and ASP.NET Core Web Application
+
+Visual Studio will show a list with Templates for Web Projects. For our example you must select **Web API,** that means you will create an ASP.NET Web API Application.
+
+Verify that you have selected ASP.NET Core 2.1 as the framework. The .NET Core 2.1 is included in the last version of Visual Studio 2017 and is automatically installed and configured for you when you install Visual Studio 2017.
+
+![Visual Studio dialog for selecting the type of an ASP.NET Core Web Application with API option selected.](media/create-web-api-application.png)
+
+**Figure 4-37**. Selecting ASP.NET CORE 2.1 and Web API project type
+
+If you have any previous version of .NET Core, you can download and install the 2.1 version from [https://www.microsoft.com/net/download/core#/sdk](https://www.microsoft.com/net/download/core#/sdk)
+
+You can add Docker support when creating the project in the previous step, or later, so you can "Dockerize" your project any time. To add the Docker support after the project creation, right-click on the solution file and select **Add > Docker support** on the context menu.
+
+![Context menu option to add Docker support to a existing project: Right click (on the project) > Add > Docker Support.](media/add-docker-support-to-project.png)
+
+**Figure 4-38**. Adding Docker support to existing project
+
+To complete adding Docker support, you have the choice of Windows or Linux, in this case select **Linux**, because AKS doesn’t support Windows Containers (as of late 2018).
+
+![Option dialog to select Target OS for Dockerfile.](media/select-linux-docker-support.png)
+
+**Figure 4-39**. Selecting Linux Containers.
+
+With these simple steps, you will have your ASP.NET Core 2.1 application running on a Linux Container.
+
+As you can see the, integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
+
+Now you can run your application with the key **F5** or using the Play Button
```

Comment:
This is missing a period and we don't usually give options. My suggestion:
Press **F5** to build and run your application.